java/src/plume/LimitedSizeIntSet.java

```diff
@@ -106,7 +106,10 @@ public void addAll(LimitedSizeIntSet s) {
     // https://tinyurl.com/cfissue/984 is fixed.
     int[] svalues = s.values;
     for (int i = 0; i < s.size(); i++) {
-      add(svalues[i]);
+      @SuppressWarnings(
+          "index") // svalues is always shorter than or equal to the length of s.size()
```

Comment:
The opposite fact is true.  `svalues` is the internal representation of `s`, but `s` does not necessarily use all slots of `svalues`, and `s.size()` reflects how much of `svalues` is used.

Also, "the length of s.size()" is redundant.  It's the size of s, or s.size().

Reply:
I've updated the comment, if you're willing to take another look